heron/tools/cli/src/python/execute.py

```diff
@@ -63,14 +63,17 @@ def heron_class(class_name, lib_jars, extra_jars=None, args=None, java_defines=N
   heron_env['HERON_OPTIONS'] = opts.get_heron_config()
 
   # print the verbose message
-  Log.debug('$> %s' % ' '.join(all_args))
-  Log.debug('Heron options: %s' % str(heron_env["HERON_OPTIONS"]))
+  Log.debug('Invoking class using command:\n``%s\'\'', ' '.join(all_args))
```

Comment:
instead of escaping single quotes it's more readable to use double quotes for the python string so you don't need to escape.